src/core_ocean/shared/mpas_ocn_diagnostics.F

```diff
@@ -175,7 +171,11 @@ subroutine ocn_diagnostic_solve(dt, statePool, forcingPool, meshPool, scratchPoo
       activeTracers(indexSalinity,:,nCells+1) = -1e34
 
 #ifdef MPAS_OPENACC
-      !$acc update device(normalVelocity)
+      !$acc enter data copyin (layerThickness, normalVelocity, &
+      !$acc                    activeTracers, ssh, seaIcePressure, &
```

Comment:
As far as I can tell, `seaIcePressure` and `surfacePressure` are never used on the device.  Are we including them for future use?

Reply:
Yes - thought we should add them for consistency now and then we can rely on them being there in future mods. I know it comes at a cost in the short term.